salt/modules/guestfs.py

```diff
@@ -71,3 +72,23 @@ def mount(location, access='rw', root=None):
     cmd = 'guestmount -i -a {0} --{1} {2}'.format(location, access, root)
     __salt__['cmd.run'](cmd, python_shell=False)
     return root
+
+
+def umount(name, disk=None):
+    '''
+    Unmount an image
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' guest.unmount /mountpoint disk=/srv/images/fedora.qcow
+    '''
+    cmd = 'guestunmount {0}'.format(name)
```

Comment:
Maybe it's also a good idea to put `-q` argument here since we're not handling the output.

Reply:
right, tough `guestunmount` is not the most verbose tool :)

Reply:
fixed